gapic-generator/bin/protoc-gen-ruby_gapic

```diff
@@ -15,19 +15,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-$LOAD_PATH.unshift ::File.expand_path("../../lib", __FILE__)
+$LOAD_PATH.unshift ::File.expand_path("../lib", __dir__)
 require "gapic/runner"
 require "google/protobuf/compiler/plugin.pb"
 
-# Ensure that no encoding conversions are done on STDIN and STDOUT
+# Ensure that no encoding conversions are done on stdin and stdout
```

Comment:
variable names may have changed, but STDIN and STDOUT are still abbreviations that should be capitalized in comments, I think

Reply:
Reverted.